hrui/CRM/HRUI/Upgrader/Steps/4706.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+trait CRM_HRUI_Upgrader_Steps_4706 {
+
+  /**
+   * Adds "is applying for NI/SSN" field
+   */
+  public function upgrade_4706() {
+    $this->up4706_create_ssn_applying_field();
+    $this->up4706_create_activity_type();
+
+    return TRUE;
+  }
+
+  /**
+   * @throws Exception
+   */
+  private function up4706_create_ssn_applying_field() {
+    $groupName = 'Inline_Custom_Data';
+    $customGroup = civicrm_api3('CustomGroup', 'get', ['name' => $groupName]);
+
+    if ($customGroup['count'] != 1) {
+      throw new \Exception("Cannot continue without Inline_Custom_Data group");
+    }
+
+    $customGroup = array_shift($customGroup['values']);
+
+    $fieldName = 'has_applied_for_identification';
+    $customField = civicrm_api3('CustomField', 'get', ['name' => $fieldName]);
+
+    if ($customField['count'] > 0) {
+      return;
+    }
+
+    // Add "is applying" field
+    $fieldData = [
+      'custom_group_id' => $customGroup['id'],
+      'name' => $fieldName,
+      'label' => 'Applied for NI/SSN',
+      'html_type' => 'Radio',
+      'data_type' => 'Boolean',
+      'weight' => 2,
```

Comment:
The weight here, How do we know that 2 is the next weight to be added, or is there a function to get the max weight id and use it here or it's not necessary?

Reply:
I don't think it's necessary, right now there are no specs for what position this should be displayed in the user profile, only for its use in the onboarding webform. I can remove weight and leave it to the default.

Reply:
Ok, think we should leave to default then.